config/schedule.rb

```diff
@@ -2,10 +2,6 @@
 
 set :output, '/dev/null'
 
-every 1.day do
```

Comment:
Running it daily incurs the cost of the `find` command, but I don't think that's a big deal. The real question is whether the code handles unchanged `public` files correctly. So, I think it'd be another issue to fire off to at least verify that it does.